content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -78,15 +83,18 @@ that case, the documentation for the signer should specify the meaning of
 additional certificates; for example, this might be the certificate plus
 intermediates to be presented during TLS handshakes.
 
-The PKCS#10 signing request format doesn't allow to specify a certificate
+The PKCS#10 signing request format does not allow to specify a certificate
 expiration or lifetime. The expiration or lifetime therefore has to be set
-through e.g. an annotation on the CSR object. While it's theoretically
-possible for a signer to use that expiration date, there is currently no
-known implementation that does. (The built-in signers all use the same
-`ClusterSigningDuration` configuration option, which defaults to 1 year,
-and can be changed with the `--cluster-signing-duration` command-line
-flag of the kube-controller-manager.)
+through the `spec.durationHint` field of the CSR object. The built-in signers
+use the `ClusterSigningDuration` configuration option, which defaults to 1 year,
+(the `--cluster-signing-duration` command-line flag of the kube-controller-manager)
+as the default when no `spec.durationHint` is specified.  When `spec.durationHint`
+is specified, the minimum of `spec.durationHint` and `ClusterSigningDuration` is
+used.
 
+{{< note >}}
+The `spec.durationHint` field was added in Kubernetes v1.22.  Consequently, earlier versions of Kubernetes will not honor this field.
```

Comment:
```suggestion
The `spec.durationHint` field was added in Kubernetes v1.22. Earlier versions of Kubernetes do not honor this field.
```
What happens if you try to set that field in a CSR submitted to an older API server?

Reply:
Like all new fields added in newer Kubernetes versions, older API servers will silently drop the field.

Reply:
> Like all new fields added in newer Kubernetes versions, older API servers will silently drop the field.

I recommend explaining that in the text: we shouldn't assume readers are closely familiar with Kubernetes API versioning rules.